frontend/src/components/Repository.vue

```diff
@@ -358,22 +358,6 @@ export default {
     }
   },
   methods: {
-    async getIntegratedModels() {
-      try {
-        const urlId = this.$route.params.model_id;
-        if (urlId) {
-          const urlIntegrateModel = this.integratedModels.find(m => m.short_name === urlId);
-          if (urlIntegrateModel) {
-            this.showIntegratedModelData(urlIntegrateModel);
-          } else {
-            // concurrence getModels api query
-            this.getModelData(urlId);
-          }
-        }
-      } catch {
-        this.errorMessage = messages.unknownError;
-      }
-    },
```

Comment:
Could you please make sure this removal doesn't hinder accessing `/gems/repository/Human-GEM`? Going to the Repository page directly and clicking on a model works, but going to the URL directly does not seem to work. If it was a problem on main, please report this new issue.